.github/workflows/check-master.yml

```diff
@@ -34,14 +34,15 @@ jobs:
         with:
           auto-update-conda: true
           python-version: ${{ matrix.python-version }}
+          miniconda-version: "latest"
 
       - name: Install latest conda
         run: |
           conda update -n base -q conda
 
       - name: Install dependencies
         run: |
-          conda install python=${{ matrix.python-version }} pip --file requirements.txt --file requirements-extra.txt
+          conda install -c defaults -c anaconda-cloud/label/dev python=${{ matrix.python-version }} pip --file requirements.txt --file requirements-extra.txt
```

Comment:
these dev channels can be dropped and at least use `-c anaconda-cloud`

Reply:
:thumbsup: